src/renderer/components/DataSettings.vue

```diff
@@ -1114,7 +1101,6 @@ async function importPlaylists() {
             return x.videoId === video.videoId && x.playlistItemId === video.playlistItemId
           })
         } else {
-          // Older playlist exports have no `playlistItemId` but have `timeAdded`
```

Comment:
There are still older playlist export files around even code for `Export Playlists For Older FreeTube Versions` removed...